radio/src/opentx.h

```diff
@@ -1385,6 +1387,10 @@ union ReusableBuffer
 #endif
   } modelsel;
 
+  struct {
+    char msg[64];
+  } msgbuf;
```

Comment:
Okay, I am still not getting it. On ARM, modelsel only has  menu_bss, which is only used in sdcard.cpp, and mainname. Since mainname isn't guaranteed to be large enough and it is cleaner I added a dedicated msg[64] struct to the union.